src/Bridge/Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -283,12 +283,24 @@ protected function addJoinsForNestedProperty(string $property, string $rootAlias
             $resourceClass = null;
         }
 
+        if (\func_num_args() > 5) {
+            $joinType = func_get_arg(5);
+        } else {
+            if (__CLASS__ !== \get_class($this)) {
+                $r = new \ReflectionMethod($this, __FUNCTION__);
+                if (__CLASS__ !== $r->getDeclaringClass()->getName()) {
+                    @trigger_error(sprintf('Method %s() will have a sixth `$joinType` argument in version API Platform 3.0. Not defining it is deprecated since API Platform 2.3.', __FUNCTION__), E_USER_DEPRECATED);
```

Comment:
Actually we should not introduce a deprecation in a patch release. We should add the deprecation error in a separate PR targeting `master`. WDYT?